src/utils/cloneWithProps.js

```diff
@@ -24,7 +24,7 @@ var CHILDREN_PROP = keyOf({children: null});
  * Sometimes you want to change the props of a child passed to you. Usually
  * this is to add a CSS class.
  *
- * @param {object} child child component you'd like to clone
+ * @param {object} child child ReactElement you'd like to clone
```

Comment:
I suppose this should be `@param {ReactElement} element`, and the same for the return type below